src/main/java/de/tum/in/www1/artemis/service/connectors/pyris/PyrisJobService.java

```diff
@@ -0,0 +1,124 @@
+package de.tum.in.www1.artemis.service.connectors.pyris;
+
+import java.security.SecureRandom;
+
+import jakarta.annotation.PostConstruct;
+import jakarta.servlet.http.HttpServletRequest;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.map.IMap;
+
+import de.tum.in.www1.artemis.service.connectors.pyris.job.PyrisJob;
+import de.tum.in.www1.artemis.web.rest.errors.AccessForbiddenException;
+
+@Service
+@Profile("iris")
+public class PyrisJobService {
```

Comment:
please add JavaDoc what the purpose of this service is. To me this is unclear. Feel free to ask ChatGPT for help.